index.html

```diff
@@ -24,35 +24,50 @@ <h1>BYOJS (Bring your own JS)</h1>
 <p><strong>BYOJS</strong> aims to champion this approach, and celebrate great web development practices.</p>
 <ul id="project-list">
 	<li><a href="https://github.com/byojs/storage" aria-label="Storage project">
-		<h3>Storage</h3>
+		<h3>
+			<img src="/img/storage.svg" width="30" height="30" />
```

Comment:
probably should have `alt` tags (at a minimum) on these `<img>` elements, no?

Reply:
Good spot.